frontend/src/pages/RecipientRecord/pages/ViewCommunicationLog/components/DisplayNextSteps.js

```diff
@@ -2,16 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ReadOnlyField from '../../../../../components/ReadOnlyField';
 
-export default function DisplayNextSteps({ title, steps }) {
+export const skipDisplaySteps = (steps) => {
```

Comment:
Function to eliminate empty next steps sections from the "view communication log" page